submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submission/SubmissionService.kt

```diff
@@ -32,17 +32,15 @@ class SubmissionService(
         return submitter.handleRequest(accNo, version)
     }
 
-    suspend fun submitAsync(rqt: SubmitRequest): SubmissionId {
+    suspend fun submitAsync(rqt: SubmitRequest) {
```

Comment:
why are we removing the return type? this is actually used by api user to know which version was the one being submitted.